firefox-ios/firefox-ios-tests/Tests/ClientTests/TabTray/TabsPanelStateTests.swift

```diff
@@ -264,21 +349,43 @@ final class TabPanelStateTests: XCTestCase {
         return TabsPanelState(windowUUID: .XCTestDefaultUUID)
     }
 
+    private func createOneSelectedTab() -> [TabModel] {
+        return [
+            MockTabModel(tabTitle: "Tab 0"),
+            MockTabModel(isSelected: true, tabTitle: "Tab 1"),
+            MockTabModel(tabTitle: "Tab 2")
+        ]
+    }
+
     private func createTabs(count: Int = 3, isPrivate: Bool = false) -> [TabModel] {
-        var tabs = [TabModel]()
-        for index in 0..<count {
-            let tab = TabModel.emptyState(tabUUID: createTabUUID(), title: "Tab\(index)", isPrivate: isPrivate)
-            tabs.append(tab)
+        return (0 ..< count).map { index in
+            MockTabModel(isPrivate: isPrivate, tabTitle: "Tab\(index)")
```

Comment:
Nice, this looks cleaner! 👌 

Is there a reason we cannot continue to use the preexisting `TabModel.emptyState()` function helper? It seems to accomplish the same thing here. I would like to continue using that instead of a new `MockTabModel()` function. Or at the very least, let's rename `MockTabModel` to follow swift function naming best practices. e.g. `generateTabModel` with lowercase first letter. I would hesitate to use the work `mock` in this function name as well since we do have a number of mock types in our project (e.g. MockProfile, MockRouter, etc.), but this function is in fact returning a standard (non-mocked) TabModel.

Reply:
Thanks for the suggestion, I somehow missed that method.